api.bs

```diff
@@ -676,18 +676,17 @@ for=PrivateAttribution>measureConversion()</a> method to find matching
 
 ### Contents ### {#impression-store-contents}
 
-The [=impression store=] must store the following information:
+The [=impression store=] must store the following information per [=impression=]:
 
-<div link-for=PrivateAttribution>
+<div dfn-for=impression link-for=PrivateAttribution>
 <pre class=simpledef>
-Filter Data: The {{PrivateAttributionConversionOptions/filterData}} value passed to <a>saveImpression()</a>.
-Impression Site: The site that called <a>saveImpression()</a>.
-Intermediary Site: The site corresponding to the script that called <a>saveImpression()</a>.
-Conversion Sites: The conversion site(s) that were passed to <a>saveImpression()</a>.
-Timestamp: The time at which <a>saveImpression()</a> was called.
-Lifetime: The number of days an [=/impression=] remains eligible for attribution,
-Lifetime: either from the call to <a>saveImpression()</a>, or a [=/user agent=]-defined limit.
-Histogram Index: The histogram index passed to <a>saveImpression()</a>.
+<dfn>Filter Data</dfn>: The {{PrivateAttributionConversionOptions/filterData}} value passed to <a>saveImpression()</a>.
+<dfn>Impression Site</dfn>: The site that called <a>saveImpression()</a>.
+<dfn>Intermediary Site</dfn>: The site corresponding to the script that called <a>saveImpression()</a>.
+<dfn>Conversion Sites</dfn>: The conversion site(s) that were passed to <a>saveImpression()</a>.
+<dfn>Timestamp</dfn>: The time at which <a>saveImpression()</a> was called.
+<dfn>Lifetime</dfn>: The number of days an [=/impression=] remains eligible for attribution, either from the call to <a>saveImpression()</a>, or a [=/user agent=]-defined limit.
+<dfn>Histogram Index</dfn>: The histogram index passed to <a>saveImpression()</a>.
```

Comment:
Can we define a numeric type for this? I think its an `unsigned long long` so this should be a 32 bit unsigned integer.